packages/web/src/root.tsx

```diff
@@ -20,13 +21,16 @@ const Root: React.FC<Props> = ({ store, history }) => (
     <ConnectedRouter history={history}>
       <React.StrictMode>
         <Switch>
-          <Route path={"/login"}>LOGIN</Route>
+          <CompatRoute path="/login">LOGIN</CompatRoute>
           <Route path="/">
             <ConfigsProvider configJson={{}}>
               <App />
             </ConfigsProvider>
           </Route>
-          <Route>NOT FOUND</Route>
+          {/* <CompatRoute path="/" configJson={{}}>
+            <App />
+          </CompatRoute> */}
+          <CompatRoute>NOT FOUND</CompatRoute>
         </Switch>
```

Comment:
@auan369  I think we can remove the router from this file `root.tsx` we are not using the route /login , so we can remove all the stuff. The only thin is that we should probably handle 404 route somehow differently? But we can resolve that when we move to react router 6

Reply:
Okay will work on this